proto/server/configEvtHandler.go

```diff
@@ -210,6 +215,28 @@ func handleNetworkSlicePost(configMsg *configmodels.ConfigMessage, subsUpdateCha
 	if errPost != nil {
 		logger.DbLog.Warnln(errPost)
 	}
+	if factory.WebUIConfig.Configuration.SendPebbleNotifications {
+		sendPebbleNotification("canonical.com/webconsole/networkslice/create")
```

Comment:
Should not this be a configurable parameter instead of a hard-coded value?

Reply:
At this point I don't see a need for parameterizing the key, because our charmed operators will be the only user of this feature. In the future, if it becomes troublesome, we could think about building this string out of configurable elements.